front-end/src/drivers/main.jsx

```diff
@@ -29,7 +29,11 @@ class drivers extends React.Component {
 
   list () {
     const items = []
-    this.props.drivers.sort((a, b) => { return parseInt(a.id) < parseInt(b.id) }).forEach((d, n) => {
+    this.props.drivers.sort((a, b) => {
+      return a.name.localeCompare(b.name,
```

Comment:
it will be nice to move this logic in a separate helper method
```js
localeCompare(str1,str2) {
    return str1.localeCompare(
      str2 ,
      navigator.languages[0] || navigator.language, 
      { numeric: true, ignorePunctuation: true }
  )
}

```

Reply:
A number of attempts were unsuccessful after a few hours trying:

.../utils/sort_by_name.js
------------------------
export const SortByName = (a, b) => {
  return a.name.localeCompare(b.name,
    navigator.languages[0] || navigator.language,
    { numeric: true, ignorePunctuation: true })
}

.../ato/main.jsx
import SortByName from '../utils/sort_by_name'
   ...
    return this.props.atos.sort((a, b) => function () {
      SortByName(a, b)
    }).map(probe => {...

For some reason it would not perform the localCompare successfully.

Turns out that the sort function is just a few characters more than the original sort by id so it's not a big setback. Left it as:

return a.name.localeCompare(b.name, navigator.languages[0] || navigator.language, { numeric: true, ignorePunctuation: true })

Thanks.

Reply:
Resolved issue; added utility function SortByName.